projects/laji/src/app/shared/navbar/navbar.component.html

```diff
@@ -45,7 +45,7 @@
                 <a [routerLink]="['/validation'] | localize">{{ 'validation.title' | translate }}</a>
               </li>
               <li [routerLinkActive]="['active']">
-                <a [routerLink]="['/results'] | localize">{{ 'results.title' | translate }}</a>
+                <a [routerLink]="['/identification'] | localize">{{ 'identification.title' | translate }}</a>
```

Comment:
This is unrelated to this specific PR, but as a sidenote imo in the future we should avoid cluttering laji project components with stuff that is related to other projects. It would be much easier to simply have a new navbar component for each project.

Reply:
I agree. I copied this style from the iucn project but both of these projects could have their own navbar component. It needs some refactoring (as the general app component includes several features such as login, language selection and feedback) so maybe it could be its own task